tests/test_configuration_auto.py

```diff
@@ -26,6 +28,11 @@
 SAMPLE_ROBERTA_CONFIG = os.path.join(os.path.dirname(os.path.abspath(__file__)), "fixtures/dummy-config.json")
 
 
+def test_module_spec():
+    assert transformers.models.auto.__spec__ is not None
+    assert importlib.util.find_spec("transformers.models.auto") is not None
```

Comment:
Please use `self.assertIsNotNone` here.

Reply:
Since this is not part of a unittest.TestCase I can't

Reply:
It should be in the `AutoConfigTest`.

Reply:
I can and will do that! I just oriented my changes on #13321 which also uses the pure pytest variant.

Reply:
Yes, I didn't catch the test was not up to our usual standards in that one :-) Thanks for fixing here!